src/other/feedback/feedback-fr.hbs

```diff
@@ -179,7 +179,12 @@
 			</section>
 		</section>
 
-		<input type="submit" name="fbsbmt" id="fbsbmt" value="Soumettre la rétroaction" class="btn btn-primary" /> <input type="reset" value="Réinitialiser" class="btn btn-default" />
+		<div class="form-group">
+			<input type="submit" name="fbsbmt" id="fbsbmt" value="Soumettre la rétroaction" class="btn btn-primary" />
+		</div>
+		<div class="form-group mrgn-tp-md">
+			<input type="reset" value="Réinitialiser la page aux valeurs par défaut" class="btn btn-link" style="padding: 0;"/>
```

Comment:
We should avoid inline style in our working example

Reply:
A custom class would need to be made.

Reply:
Yes that could work, We already added a similar CSS class in GCWeb documented in the [experimental page](https://wet-boew.github.io/themes-dist/GCWeb/experimental-en.html).

We can add it if it is needed and if we have a use case that require that feature. I don't want to overload the wet-boew CSS with style that is not necessary required. It will need to be documented and supported with working example (which could be in the style guide). Whenever is possible, we should try to use the same naming as the equivalent CSS class in Bootstrap 4.